Documentation/docs/appendix.md

```diff
@@ -225,9 +225,55 @@ If the SIMPLE_EXTRUSION is used, bottom topography can be optionally added to th
 		eqn = f(x,y) = some function of (x,y) as an equation
 	\end{TOPOGRAPHY}
 
-
-
-
-
-
-
+## Appendix E: ABAQUS mesh file format<a name="ABAQUS"></a>
+The ABAQUS mesh file which should use the extension `.inp` is divided into two parts. The top portion is the standard [Abaqus file format](https://abaqus-docs.mit.edu/2017/English/SIMACAEMODRefMap/simamod-c-model.htm). The skeleton of this form for a quadrilateral mesh with 122 corner nodes and 103 elements is as follows:
```

Comment:
In general, I thought it might be nice to add a sentence in the introductory part of this appendix that gives a motivation for the Abaqus file format and its custom extension, by mentioning that this additional information can be parsed in Trixi to create high-order curved AMR meshes using P4estMesh (and adding an appropriate link to either Trixi and/or our Trixi docs (although the latter is dangerous, since if we change the docs at some point, we'll surely have forgotten about  this reference here)).

Reply:
I added a short paragraph to motivate the new file format.